Sources/TSCBasic/Path.swift

```diff
@@ -544,11 +547,12 @@ private struct UNIXPath: Path {
 
     init(normalizingAbsolutePath path: String) {
       #if os(Windows)
-        var buffer: [WCHAR] = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
-        _ = path.withCString(encodedAs: UTF16.self) {
-            PathCanonicalizeW(&buffer, $0)
+        var result: [WCHAR] = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
+
+        _ = path.standardizingPathSeparator().withCString(encodedAs: UTF16.self) {
+            PathCchCanonicalize($0, result.length, $0)
         }
```

Comment:
I'm confused.  How does this work?  `result` is unused other than to compute its length?  This will just mutate the string in place, which would result in the result always being empty?

Reply:
Thanks for pointing out the bug! Fixed.